### ISSUE TITLE
a question

### ISSUE THREAD

**Issue by Pecus35** [Dasrg/steam-rateup-bot#1]
can you make an artwork favorite bot?

**Comment by Dasrg** [comment]
Hey! I added this ability to the bot. Thank you for your request

**Comment by Pecus35** [comment]
do i need steam guard for artwork favorite bot?

**Comment by Dasrg** [comment]
yes, you need enter `shared_secret` code in the `bots.txt`

**Comment by Pecus35** [comment]
okay but you can use limited accs for artwork favorite bot you should make favorite_bot.txt for artwork favorite and like_bot.txt for like bot

**Comment by Pecus35** [comment]
are you going to fix it?

**Comment by Dasrg** [comment]
Yes, now you can use limited accs without steam guard. In the config: `"limited": true`
And in the `bots.txt` : 
```
login1:password1:
login2:password2:
login3:password3:
```
with the colon in the end of lines, 
you can add it in the notepad++: ctrl+h, select `Extended`, in `find what` enter: `\r\n`, in `replace with` enter: `:\r\n`

If you have an `error: CAPTCHA` change ip or wait some time

**Comment by Pecus35** [comment]
okay thanks

**Comment by Pecus35** [comment]
can you add something like desired favourite amount?

**Comment by Dasrg** [comment]
Added the `amount` parameter in the bot. If `amount: 0` script will use number of accounts in the `bots.txt`. 

Thank you for your requests

**Comment by Pecus35** [comment]
Hey can you give me your discord? I want to discuss about something with you

**Comment by Dasrg** [comment]
DAS#1202
